File: appsflyer-uid/android/src/main/java/net/bitburst/plugins/appsflyeruid/AppsflyerUid.java
package net.bitburst.plugins.appsflyeruid;

import android.content.Context;
import com.appsflyer.AppsFlyerLib;

public class AppsflyerUid {

    public String getUID(Context context) {
        String uid = "";
        AppsFlyerLib appsflyer = AppsFlyerLib.getInstance();
        if (appsflyer != null) {
            uid = appsflyer.getAppsFlyerUID(context);
        }
        return uid;
    }
}

File: appsflyer-uid/ios/Plugin/AppsflyerUid.swift
import Foundation
import AppsFlyerLib

@objc public class AppsflyerUid: NSObject {
    @objc public func getUID() -> String {
        var uid = ""
        if(AppsFlyerLib.shared().getAppsFlyerUID() != "") {
            uid = AppsFlyerLib.shared().getAppsFlyerUID()
        }

        return uid
    }
}
